README_python.md

```diff
@@ -170,35 +200,240 @@ essentially zero cost, with storage being shared. Thus, we can usually pass it
 directly to existing Python APIs (like `imsave()`) that expect 'image-like'
 objects without any explicit conversion necessary.
 
-## Using Halide Generators from Python
+## Halide Generators In Python
+
+In Halide, a "Generator" is a unit of encapsulation for Halide code. It is
+self-contained piece of code that can: - Produce a chunk of Halide IR (in the
```

Comment:
Should this be a real list?

Reply:
jfc, the markdown formatter I use keeps un-formatting this from a real list, fixed